README.md

```diff
@@ -14,19 +14,24 @@ There is also a SignatureUtils class which simplifies the generation of the secu
 ### Requirements
 
 * PHP 5.6+
-* the `make` utility (usually available in the development tools of your platform)
 * You must be connected to the internet to use this site
 
-
 ## Getting Started
 
 The following shows you how to get the code in this repository up and running as
-a stand-alone site. First, get the code with
+a stand-alone site.
+
+First, get the code with
 
     git clone --recursive https://github.com/Learnosity/learnosity-demos.git
 
-You can then use the simple `Makefile` included with this project to just start
-one or the other modes, by running either of the following commands from the
+Note: To simplify the next steps, you'll need the `make` utility (usually available
```

Comment:
@mcgoughn @richshupe I actually bit the bullet and fixed the README file to clarify that `make` is not a hard dependency, and that the site can be run without it following other sections of the doc.

I think this should be sufficient to merge this PR without concern?